pandas/core/generic.py

```diff
@@ -8518,24 +8518,198 @@ def _doc_parms(cls):
 """
 
 _cnum_doc = """
+Return %(desc)s over a DataFrame or Series axis.
+
+Returns a DataFrame or Series of the same size containing the %(desc)s.
 
 Parameters
 ----------
-axis : %(axis_descr)s
+axis : {0 or 'index', 1 or 'columns'}, default 0
```

Comment:
This is right, `cum_func` (i.e. function corresponding to all cumulative methods) is defined with `axis=None` as default argument.

I also found #20289 as a PR with the correct axis parameter.

Reply:
Although it is technically None, in practice it is 0 for Series/DataFrame, so I would keep the documentation like this. 
The technical reason is because for Panel it is 1, but Panel is deprecated and I think we should not care about them in the documentation.